scripts/update/Updater.php

```diff
@@ -297,6 +297,6 @@ public function update($initialVersion) {
             $this->setVersion('6.9.0');
         }
 
-        $this->skip('6.9.0', '6.15.0');
+        $this->skip('6.9.0', '6.15.1');
```

Comment:
Changing the interface can cause breaking changes, some that I spotted:
* tao_models_classes_service_StateStorage::set()
* ServiceManager::register()
* ExtendedState::save()
* StorageManager::persistCacheEntry()